ui/package.json

```diff
@@ -38,7 +38,6 @@
     "@tremor/react": "^3.6.6",
     "@types/event-source-polyfill": "^1.0.1",
     "@types/jest": "^28.1.2",
-    "@types/js-yaml": "^4.0.5",
```

Comment:
This PR is now removing the package `@types/js-yaml` but funny enough, it is not in use anymore*, so it is good that we remove it! 

*_I think we must have use `js-yaml` package in a previous version, not 100% sure_